lambda/src/main/scala/pricemigrationengine/services/EngineSecrets.scala

```diff
@@ -0,0 +1,56 @@
+package pricemigrationengine.services
+
+import pricemigrationengine.model._
+import zio.{Layer, ZIO, ZLayer}
+import software.amazon.awssdk.regions
+import software.amazon.awssdk.services.secretsmanager._
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest
+import upickle.default._
+
+import java.lang.System.getenv
+
+case class EngineSecrets(
+    zuoraApiHost: String,
+    zuoraClientId: String,
+    zuoraClientSecret: String,
+    salesforceClientId: String,
+    salesforceClientSecret: String,
+    salesforceUserName: String,
+    salesforcePassword: String,
+    salesforceToken: String,
+    salesforceAuthUrl: String
+)
+
+object EngineSecrets {
+
+  implicit val reader: Reader[EngineSecrets] = macroRW
+
+  private lazy val region: regions.Region = regions.Region.EU_WEST_1
+
+  private lazy val secretsClient = SecretsManagerClient.create()
+
+  private lazy val secretId: String = s"price-migration-engine-lambda-${getenv("stage")}"
```

Comment:
`getenv` is really an effect and there's a ZIO wrapper for it: https://zio.dev/reference/services/system/#system-environment

Reply:
Nice one. I was wondering about that...

Reply:
Done here: https://github.com/guardian/price-migration-engine/pull/852/commits/be8cb7e5f55bbbb1e177b00bd68db916379adb7d